runtime.md

```diff
@@ -34,22 +34,23 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is started to when it is stopped or crashes unexpectedly.
+
+1. OCI compliant runtime is invoked by passing the bundle path as argument.
+2. The container's runtime environment is created.
+3. The container's state is written to the filesystem.
```

Comment:
On Wed, Oct 28, 2015 at 07:37:11AM -0700, Doug Davis wrote:
> > +3. The container's state is written to the filesystem.
> 
> Let's be specific and say where the `state.json` file resides -
> e.g. `/run/opencontainers/runc/<containerID>/state.json`

There is no specific location for non-Linux OSes at the moment.  See
[1] (in an outdated diff on #211), where @vbatts says we're waiting
for PoC implementations on other OSes before specifying paths for
them.  And there's been recent talk about making this write optional
as well, to support users who don't have permissions to write to a
system-wide directory.

[1]: https://github.com/opencontainers/specs/pull/211#discussion_r41067134